doc/tutorials/biicode/tutorial_biicode/tutorial_biicode.markdown

```diff
@@ -0,0 +1,121 @@
+Using OpenCV with biicode dependency manager
+========================================
+
+[**biicode**](http://opencv.org/biicode.html) is a solution for resolving and keeping track of dependencies and version compatibilities for C/C++ projects. 
+
+Using the *"hooks” feature* of biicode, **getting started with OpenCV in C/C++** is very simple. Just write an `#include` to opencv headers and biicode will manage to retrieve and install OpenCV in your computer and configure your project.
+
+As simple as that, that's why it is a great tool to use OpenCV in an easy way.
+
+In the following lines, we'll try to describe how to get started with OpenCV using [biicode](http://www.biicode.com/).
```

Comment:
más directo, yo pasaría del In the following lines...  a "How to get started with OpenCV using biicode"